bot/cogs/help_channels.py

```diff
@@ -587,6 +595,17 @@ async def on_message(self, message: discord.Message) -> None:
             return  # Ignore messages sent by bots.
 
         channel = message.channel
+        if self.is_in_category(channel, constants.Categories.help_in_use):
+            if channel.id in self.unanswered:
+                claimant_id = self.help_channel_claimants[channel].id
+
+                if claimant_id != message.author.id:
+                    self.unanswered[channel.id] = False
+
+                    await channel.edit(
+                        name=f"{IN_USE_ANSWERED_EMOJI}{NAME_SEPARATOR}{self.get_clean_channel_name(channel)}"
+                    )
```

Comment:
This should be moved to a separate function. It will keep the function sizes more manageable and make it easier to write tests down the road.